xml/config/apps/demo/Gui3LeftTree.xml

```diff
@@ -8,18 +8,22 @@
 >
 
   <name>My Layout</name>
+    <name xml:lang="cs">Moje rozvržení</name>
```

Comment:
Right.  That needs a schema change to permit it.  I hadn't noticed that was missing until now.  I'll get that done, then pull the WIP tag from this.